lib/ansible/plugins/action/__init__.py

```diff
@@ -841,13 +841,9 @@ def _update_module_args(self, module_name, module_args, task_vars):
         # make sure modules are aware if they need to keep the remote files
         module_args['_ansible_keep_remote_files'] = C.DEFAULT_KEEP_REMOTE_FILES
 
-        # make sure all commands use the designated temporary directory if created
-        if self._is_become_unprivileged():  # force fallback on remote_tmp as user cannot normally write to dir
-            module_args['_ansible_tmpdir'] = None
```

Comment:
this should not be None, it should already have resolved to the configured system_dirs, so should be changed anyways